<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationInfo</class>
 <widget class="QWidget" name="RegistrationInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>员工信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="userid_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>名字：</string>
   </property>
  </widget>
  <widget class="QLabel" name="sex_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="age_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLabel" name="job_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工作：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userid_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="sex_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="job_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_ok_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>190</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
